freeipa/src/main/java/com/sequenceiq/freeipa/sync/StackStatusCheckerJob.java

```diff
@@ -113,7 +115,11 @@ private void updateStackStatus(Stack stack, SyncResult result, List<ProviderSync
         DetailedStackStatus status = getStackStatus(providerSyncResults, result, alreadyDeletedCount);
         if (status != stack.getStackStatus().getDetailedStackStatus()) {
             if (autoSyncConfig.isUpdateStatus()) {
-                stackUpdater.updateStackStatus(stack, status, result.getMessage());
+                if (flowLogService.isOtherFlowRunning(stack.getId())) {
+                    throw new InterruptSyncingException(":::Auto sync::: interrupt syncing, flow is running on freeipa stack " + stack.getName());
```

Comment:
could you please add a more accurate message? when did happened or stuff like this

Reply:
you will see it from stack trace ;) but I will add some custom thing here